applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -44,6 +44,55 @@ const useStyles = makeStyles({
       },
     },
 
+    '& .sidebar-label': {
```

Comment:
@paularah i would be better option to name classes in relation to 'population' rather than sidebar cause this isn't being reused anywhere else in the 'sidebar'. In my option either its better to use class names which describes either its html element or use or in relation to what UI element